packages/convai-widget-core/src/contexts/session-config.tsx

```diff
@@ -35,6 +38,9 @@ export function SessionConfigProvider({
     tts: {
       voiceId: overrideVoiceId.value,
     },
+    conversation: {
+      textOnly: parseBoolAttribute(overrideTextOnly.value) ?? undefined,
```

Comment:
Double-checking - this will be overridden if user start the convo by text even if set to false, right?